lib/src/screens/dashboard/widgets/menu_widget.dart

```diff
@@ -35,6 +35,7 @@ class MenuWidgetState extends State<MenuWidget> {
       this.bitcoinCashIcon = Image.asset('assets/images/bch_icon.png'),
       this.decredIcon = Image.asset('assets/images/decred_menu.png');
 
+
```

Comment:
2 empty line spaces necessary?

Reply:
I think I changed this in the last pr unintentionally. The diff with master should make more sense.

```
$ git diff main -- lib/src/screens/dashboard/widgets/menu_widget.dart
diff --git a/lib/src/screens/dashboard/widgets/menu_widget.dart b/lib/src/screens/dashboard/widgets/menu_widget.dart
index bf001eb3..2e81d4cd 100644
--- a/lib/src/screens/dashboard/widgets/menu_widget.dart
+++ b/lib/src/screens/dashboard/widgets/menu_widget.dart
@@ -32,7 +32,8 @@ class MenuWidgetState extends State<MenuWidget> {
       this.ethereumIcon = Image.asset('assets/images/eth_icon.png'),
       this.nanoIcon = Image.asset('assets/images/nano_icon.png'),
       this.bananoIcon = Image.asset('assets/images/nano_icon.png'),
-      this.bitcoinCashIcon = Image.asset('assets/images/bch_icon.png');
+      this.bitcoinCashIcon = Image.asset('assets/images/bch_icon.png'),
+      this.decredIcon = Image.asset('assets/images/decred_menu.png');
 
 
   final largeScreen = 731;
@@ -54,6 +55,7 @@ class MenuWidgetState extends State<MenuWidget> {
   Image bitcoinCashIcon;
   Image nanoIcon;
   Image bananoIcon;
+  Image decredIcon;
 
   @override
   void initState() {
@@ -219,6 +221,8 @@ class MenuWidgetState extends State<MenuWidget> {
         return nanoIcon;
       case WalletType.banano:
         return bananoIcon;
+      case WalletType.decred:
+        return decredIcon;
       default:
         throw Exception('No icon for ${type.toString()}');
     }
```